src/Graph.ts

```diff
@@ -454,6 +456,15 @@ export default class Graph {
 			module.dependencies.forEach(depModule => {
 				if (!depModule.isExternal) {
 					visit(<Module>depModule, seen);
+					const depId = relativeId(depModule.id);
+					cycles.forEach(cycle => {
+						if (cycle.path[depId] === null) {
+							cycle.path[depId] = modId;
```

Comment:
`modId` can just be recomputed.

Reply:
Alternatively we could do the relative pathing in the warning function itself as a map.

Reply:
Ok. I tried doing it in the warning function, but seemed more complex. I can try again, though.

Reply:
Ok, it's actually pretty straightforward.